### PR TITLE
Added shortner button disability check

### PR DESCRIPTION
**Pull request by LalitKushwah**

### CODE REVIEW

**Comment by origranot** [comment]
Hey, I would prefer to use a state to store the input.

Please check out: https://qwik.builder.io/docs/components/state/

File: frontend/src/routes/index.tsx [hunk 1]
```diff
@@ -115,6 +115,16 @@ export default component$(() => {
       handleShortenerClick$();
     }
   });
+  
+const changeBtnVisibility$ = $((e: Event) => { 
+  const inputValue = e?.target?.value;
+  const shortnerBtn = document?.getElementById("shortenerBtn");
```
Comment (origranot) [inline frontend/src/routes/index.tsx:121]: Hey, I would prefer to use a state to store the input.

Please check out: https://qwik.builder.io/docs/components/state/

**Review by origranot — CHANGES_REQUESTED** [review]

File: frontend/src/routes/index.tsx (hunk 1 above)
Comment (LalitKushwah) [reply frontend/src/routes/index.tsx]: @origranot Made suggested change please review again

**Comment by origranot** [comment]
Great!

**Pull request merged**